### PR TITLE
LP-2627 - Add view and register url for our team page

### PR DESCRIPTION
**Pull request by MunamMubashir-arbisoft**
[LP-2627
](https://philanthropyu.atlassian.net/browse/LP-2627)
- Remove our team page from static pages as it doesn't cater dynamic content
- Register a url in our_team app
- Add a view for our_team page
- Fix footer link for `our_team`
- Fix unit tests

[Theme PR](https://github.com/OmnipreneurshipAcademy/adg-edx-theme/pull/238)

### CODE REVIEW

**Comment by zahid-arbisoft** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1616/workflows/workflows_id/jobs/1616/artifacts)**. Coverage !!! :scream:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/branding_extension/helpers.py (100%)
openedx/adg/lms/branding_extension/tests/helpers_tests.py (100%)
openedx/adg/lms/our_team/urls.py (100%)
openedx/adg/lms/our_team/views.py (50.0%): Missing lines 17,19-20,22-23
-------------
Total:   15 lines
Missing: 5 lines
Coverage: 66%
-------------
```

**Comment by zahid-arbisoft** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1617/workflows/workflows_id/jobs/1617/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/branding_extension/helpers.py (100%)
openedx/adg/lms/branding_extension/tests/helpers_tests.py (100%)
openedx/adg/lms/our_team/tests/test_views.py (100%)
openedx/adg/lms/our_team/urls.py (100%)
openedx/adg/lms/our_team/views.py (100%)
-------------
Total:   34 lines
Missing: 0 lines
Coverage: 100%
-------------
```

File: lms/envs/common.py [hunk 1]
```diff
@@ -2746,7 +2746,6 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 
     # Verified Certificates
     'WHAT_IS_VERIFIED_CERT': None,
-    'OUR_TEAM': 'our_team',
```
Comment (harris-irfan) [inline lms/envs/common.py:2749]: If this change was already added in core changes, please remove it from there.

**Review by harris-irfan — APPROVED** [review]
LGTM 👍

File: lms/envs/common.py (hunk 1 above)
Comment (MunamMubashir-arbisoft) [reply lms/envs/common.py:2749]: I'll take a look and update it. Thanks for pointing out!

**Review by annas-usman — APPROVED** [review]
LGTM, although why are we making a new url and view?

**Comment by MunamMubashir-arbisoft** [comment]
@annas-usman The static pages are very aggressively cached and are therefore not suitable for any kind of dynamic content.

**Pull request merged**